cg/server/api.py

```diff
@@ -540,7 +543,7 @@ def get_delivery_message_for_order(order_id: int):
         response_dict: dict = response.model_dump()
         return make_response(response_dict)
     except OrderNotDeliverableError as error:
-        return make_response(jsonify(error=str(error)), HTTPStatus.PRECONDITION_FAILED)
+        return make_response(jsonify(message=str(error)), HTTPStatus.NO_CONTENT)
```

Comment:
Seems as if NO_CONTENT results in no message either way, so maybe just remove it?

Reply:
Like empty string it or something.

Reply:
Makes sense to have nothing returned in this case except the status